parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -624,7 +659,7 @@ impl snowbridge_ethereum_beacon_client::Config for Runtime {
 	type MaxFinalizedHeaderSlotArray = MaxFinalizedHeaderSlotArray;
 	type ForkVersions = ChainForkVersions;
 	type WeakSubjectivityPeriodSeconds = WeakSubjectivityPeriodSeconds;
-	type WeightInfo = weights::snowbridge_ethereum_beacon_client::WeightInfo<Runtime>;
+	type WeightInfo = ();
```

Comment:
Is this removed just for benchmark purposes? I think this needs to be here for release as this line uses the beacon client weights generated by the benchmark-ci.sh process.